feature-libs/cart/added-to-cart-toast/added-to-cart-toast.module.ts

```diff
@@ -1,5 +1,5 @@
 import { NgModule } from '@angular/core';
-import { AddedToCartToastComponentsModule } from '../components/added-to-cart-toast-components.module';
+import { AddedToCartToastComponentsModule } from './components';
```

Comment:
Please use this approach:

import { AddedToCartToastComponentsModule } from '@spartacus/cart/added-to-cart-toast/components';